lib/transport/selenium-webdriver/service-builders/chrome.js

```diff
@@ -46,6 +46,11 @@ class ChromeServiceBuilder extends BaseService {
 
     return super.createService();
   }
+
+
+  get requiresDriverBinary() {
```

Comment:
I believe the solution is effective, but it may be necessary to eliminate this check since it is no longer necessary now that Selenium WebDriver supports Selenium Manager.https://github.com/nightwatchjs/nightwatch/blob/187835f2cd3839f074c0e58f093d9993e42053f1/lib/transport/selenium-webdriver/options.js#L276-L288

Reply:
This will be useful when people need to use custom driver versions. So keeping this

Reply:
In the case of custom driver versions users might provide it in the nightwatch config itself. How would this will be useful?

Reply:
Yes, but this gives user the ability to do `npm install chromedriver@version`
No need to manually download and set driver path.

Reply:
Does selenium manager downloads driver according to the browser version?

Reply:
If that is the case then I don't see a use case to keep this. We anyway can't use different versions of driver with different browser versions.

Reply:
Not sure how selenium manager handles upgrades and downgrades (will check and update on this). Also not sure how chromium or brave or any other webkit based browser is handled. I think it is harmless to keep this code.

Reply:
Okay, selenium manager does take into account the current browser version and updates driver binaries accordingly. But even then we can not remove this (without breaking change) because someone might be using driver version from npm package and `cli_args` to point to browser binary. 
Selenium manager user the binary path specified in capabilities. But with the old way, it was possible to specify binaries using the `cli_args` options.